src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx

```diff
@@ -1,35 +1,37 @@
 import { ChangeEvent } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 
 type AuthorSearchInputProps = {
   text: string;
   onChange: (e: ChangeEvent<HTMLInputElement>) => void;
 };
 
-const useStyles = makeStyles((theme) => ({
-  input: {
-    fontSize: '1.6rem',
-    '&:hover': {
-      border: '1px solid',
-      borderColor: theme.palette.primary.main,
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    input: {
+      fontSize: '1.6rem',
+      '&:hover': {
+        border: '1px solid',
+        borderColor: theme.palette.primary.main,
+      },
+      '&:focus': {
+        border: '2px solid',
+        borderColor: theme.palette.primary.main,
```

Comment:
I notice when focused, the input box size increased by 2 pixels. Could you put a `box-sizing: border-box' here to make it consistent?

Reply:
Let's keep this for now. The release is tomorrow and then we will be moving to a new UI. So let's just make it functional for now.

Reply:
Cool, we can always change it later.